api/middleware/endpointThrottler.go

```diff
@@ -14,61 +14,6 @@ type throttlerGetter interface {
 	GetThrottlerForEndpoint(endpoint string) (core.Throttler, bool)
 }
 
-// CreateEndpointThrottler will create a middleware-type of handler to be used in conjunction with special
-// REST API end points that need to be better protected
-func CreateEndpointThrottler(throttlerName string) gin.HandlerFunc {
```

Comment:
Do we have something instead of this?

Reply:
yes, in previous PRs I have added `CreateEndpointThrottlerFromFacade` that remained. There are also unit tests that prove the functionality is still working

Reply:
great